### DISCUSSION TITLE
Option to use only A or B

### DISCUSSION THREAD

**Discussion by Nikseell** [jvde-github/AIS-catcher#280] [Q&A]
There are 2 modes -c AB and -c CD. I want to make so there are 2 more options -c A and -c B.

161.975 Normal AIS "A" -> ship to ship
162.025 Normal AIS "B" -> ship to shore

This filter would allow option to check only specific channel communications.Is it possible to make for the whole project including website so it only uses specific channel?

**Comment by jvde-github** [comment]
I have added a filter option, perhaps this will help. For example, if you want to only use Ch A in the webviewer:
```
AIS-catcher -t txt aiscatcher.org 4999 -N 8100 filter on allow_channel A
```

But you can add these `FILTER ON ALLOW_CHANNEL ABCD` also to -u, -o etc. 

Does that help?

**Reply by niklavs-shipprojects** [reply]
This was what I needed!

Thank your for quick response
